drivers/linstorvhdutil.py

```diff
@@ -102,20 +102,25 @@ def wrapper(*args, **kwargs):
                     response = call_remote_method(
                         self._session, host_ref_attached, remote_method, device_path, remote_args
                     )
+                    util.SMlog(
+                        "Success accessing on attached node for device {} ({}): {}: {}".format(device_path, vdi_uuid, remote_method, str(response)),
+                        priority=util.LOG_DEBUG
+                    )
                     return response_parser(self, vdi_uuid, response)
             except Exception as e:
                 util.SMlog(
-                    'Failed to call method on attached host. Trying local access... (cause: {})'.format(e),
+                    'Failed to call method on attached node for device {} ({}): {}. Trying accessing on master... (cause: {})'.format(device_path, vdi_uuid, remote_method, e),
                     priority=util.LOG_DEBUG
                 )
 
             try:
                 master_ref = self._session.xenapi.pool.get_all_records().values()[0]['master']
                 response = call_remote_method(self._session, master_ref, remote_method, device_path, remote_args)
+                util.SMlog("Success accessing on master for device {} ({}): {}: {}".format(device_path, vdi_uuid, remote_method, str(response)), priority=util.LOG_DEBUG)
                 return response_parser(self, vdi_uuid, response)
             except Exception as e:
                 util.SMlog(
-                    'Failed to call method on master host. Finding primary node... (cause: {})'.format(e),
+                    'Failed to call method on master for device {} ({}): {}. Finding primary node... (cause: {})'.format(device_path, vdi_uuid, remote_method, e),
```

Comment:
I know that was my message, but it's probably a good idea to change the "primary node" text.